core/arch/arm/include/kernel/thread.h

```diff
@@ -596,6 +596,22 @@ struct mobj *thread_rpc_alloc_payload(size_t size);
  */
 void thread_rpc_free_payload(struct mobj *mobj);
 
+/**
```

Comment:
I'd prefer this in a separate patch (in this PR in OK).

Reply:
ok